src/cmds/init/initUtils.ts

```diff
@@ -138,19 +163,22 @@ export async function buildWeb3Wallet(argv: YargsArgs): Promise<[
 
   return endWeb3Wallet();
```

Comment:
looks like this function is used before its defined

Reply:
The lint config permits hoisting for functions declared with the `function` keyword, and we do use that pattern in a variety of places throughout the codebase.

Reply:
just confusing to read because it's used before its defined and returning like that reads as if it has unreachable code.

Reply:
Since the order of the functions in this file predates / is orthogonal to the lint config, let's leave this unresolved and follow up on it in a separate issue or PR as well!